app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -602,7 +602,10 @@ export class Dispatcher {
     }
   }
 
-  /** Opens a Git repository in the user provided program */
+  /**
+   * Opens a path (can either be a folder representing a repository or the
+   * path to a file) in the external editor selected by the user.
+   */
   public async openInExternalEditor(path: string): Promise<void> {
```

Comment:
Should the `path` parameter be renamed to `fullPath` here as well since it's been renamed elsewhere?